### DISCUSSION TITLE
Is this possible to be implemented?

### DISCUSSION THREAD

**Discussion by SynStark** [seerge/g-helper#1204] [Ideas]
Acer Predator Laptops have this very easy Fan Control option controlling the RPM of the fans with a slider. I can literally hear the fans slowing down or ramping up as I move the slider. Since I'm not very savvy with the graphs and I'm guessing other people out there like me are not very good with this stuff, is it possible to insert sliders like this under Custom Fan Control or something? 

Here is a picture:
https://us.v-cdn.net/6029997/uploads/editor/g8/8bstkekxkd94.jpg

Thank you for all the hard work!

**Comment by seerge** [comment]
@SynStark no, usually you can't control fans like that. If you want them to run on max - just hold SHIFT and crank up whole curve to the top. But it's not recommended anyway

**Reply by SynStark** [reply]
That's a picture from Google sorry. I'm more interested in making them more quiet. At night when browsing, or doing something in silence I hear constant fans 1800-2200 rpm. If I set it to silent mode they completely stop but the temps rise to 70c. So If I could manually set them to 500 or 1000 they would still cool the system but just be quiet.

**Reply by seerge** [reply]
You can try to edit fan curve then manually, or just use a silent mode with default one
